README.md

```diff
@@ -62,10 +62,61 @@ module.exports = {
 };
 ```
 
+This example demonstrates how to use this plugin to minify the css assets loaded using `mini-css-extract-plugin`.
+
+```javascript
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+
+module.exports = {
+  mode: 'production',
+  entry: ['./src/style.css'],
+  optimization: {
+    minimizer: [new OptimizeCSSAssetsPlugin({
+      assetNameRegExp: /\.css$/,
+      cssProcessor: require('cssnano'),
+      cssProcessorPluginOptions: {
+        preset: ['default', { discardComments: { removeAll: true } }],
+      },
+      canPrint: true
+    })],
+  },
+  plugins: [
+    new MiniCssExtractPlugin({
+      filename: '[name].[hash].css',
+      chunkFilename: '[id].[hash].css',
+    }),
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: [
+          {
+            loader: MiniCssExtractPlugin.loader,
+            options: {
+              publicPath: (resourcePath, context) => {
+                return path.relative(path.dirname(resourcePath), context) + '/';
+              }
+            }
+          },
+          {
+            loader: 'css-loader',
+            options: {}
+          }
+        ],
+      },
+    ],
+  },
+};
+```
```

Comment:
Please remove this example. You've already put a link to the example project under `Examples` which is enough.